code/modules/vehicles/wheelchair.dm

```diff
@@ -41,11 +41,12 @@
 
 /obj/vehicle/ridden/wheelchair/driver_move(mob/living/user, direction)
 	if(istype(user))
-		if(canmove && (user.usable_hands < arms_required))
-			to_chat(user, "<span class='warning'>You don't have enough arms to operate the wheels!</span>")
-			canmove = FALSE
-			addtimer(VARSET_CALLBACK(src, canmove, TRUE), 20)
-			return FALSE
+		if(canmove && rider_check_flags & REQUIRES_ARMS)
+			if(HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
+				to_chat(user, "<span class='warning'>You don't have enough arms to operate the wheels!</span>")
+				canmove = FALSE
+				addtimer(VARSET_CALLBACK(src, canmove, TRUE), 20)
+				return FALSE
```

Comment:
```suggestion
		if(canmove && rider_check_flags & REQUIRES_ARMS && HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
			to_chat(user, "<span class='warning'>You don't have enough arms to operate the wheels!</span>")
			canmove = FALSE
			addtimer(VARSET_CALLBACK(src, canmove, TRUE), 2 SECONDS)
			return FALSE
```
Is the feedback message correct here?

Reply:
It probably needs to be clarified more specifically that you lack usable hands rather than just number of arms